galaxy/values.yaml

```diff
@@ -6,8 +6,8 @@ nameOverride: ""
 fullnameOverride: ""
 
 image:
-  repository: galaxyproject/galaxy-min
-  tag: 21.09
+  repository: galaxy/galaxy-min
```

Comment:
This will require a change in the galaxy auto-build Action, probably best to do as part of this PR https://github.com/galaxyproject/galaxy/pull/14449.

Reply:
Done